packages/common/src/types/index.ts

```diff
@@ -71,11 +92,11 @@ export type CurrentUser = {
   avatarUrl: string | undefined;
```

Comment:
@SaraVieira This can just be

```suggestion
  avatarUrl?: string;
```

Reply:
They need to be there but will return undefined if not set but they need to exist so its not actually optional

Reply:
`undefined` ('not defined') is the same like it's not there, no? 🤔